application/src/test/resources/missing_connection_delay.yml

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright 2024 Telefonaktiebolaget LM Ericsson
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
```

Comment:
I think in that case, we would have other test cases to test default value, so you can change the filename to "nothing_set.yml" and use it as the sidecar use https://github.com/Ericsson/ecchronos/blob/master/application/src/test/resources/nothing_set.yml

Reply:
Using Rajid's nothing_set.yml now

Reply:
Now using nothing_set.yml